### ISSUE TITLE
[3.1.0] [!] use_native_modules! skipped the react-native dependency 'react-native-avoid-softinput'. No podspec file was found.

### ISSUE THREAD

**Issue by efstathiosntonas** [mateusz1913/react-native-avoid-softinput#141]
### Environment

Library version: `3.1.1`
OS version: ??? 

### Affected platforms

- [ ] Android
- [X] iOS

### Current behavior

After upgrading to `3.1.0` and  `pod install` on react-native `0.68.5` it throws: 

`[!] use_native_modules! skipped the react-native dependency 'react-native-avoid-softinput'. No podspec file was found.`

### Expected behavior

`pod install` should install :)

### Reproduction

none

**Comment by mateusz1913** [comment]
@efstathiosntonas do you use manual linking?
With v3.1.0 the pod was renamed from `react-native-avoid-softinput` to `ReactNativeAvoidSoftinput` (because xcode does not like `-` chars and changes them to `_` when generating swift-objc interface).
Is it possible for you to provide reproducible repo?

**Comment by efstathiosntonas** [comment]
No manual linking, plain yarn add and pod install afterwards. Can’t create reproducible since I’m using an “old” react native version, sorry 😞

**Comment by mateusz1913** [comment]
Pls provide reproduction repo

// For RN version 0.68.x
```
npx react-native init MyApp --template react-native-template-typescript@6.10.4
```

**Comment by mateusz1913** [comment]
Or just `yarn remove` and install lib again

**Comment by efstathiosntonas** [comment]
`npx react-native init MyApp --template react-native-template-typescript@6.10.4` throws zerror Failed to load configuration of your project.`

Steps I've tried:
1. remove package
2. `pod install` (just to make sure)
3. `yarn add react-native-avoid-softinput` 
4. `pod install`
5. throws the same error

afterwards tried `pod update`, same error. 

Then I've deintegrated pods `pod deintegrate`, `pod install` again, same error.


It seems that the podspec is missing from the lib:
![Screenshot 2023-01-30 at 10 35 59](https://user-images.githubusercontent.com/717975/215427229-3d9977a4-bfc6-43c7-85cd-415d92a18efe.png)

**Comment by mateusz1913** [comment]
Got it! There's no podspec exported to npm, I'll make a fix and release quickly

**Comment by efstathiosntonas** [comment]
thanks Mateusz, keep it up!

**Comment by mateusz1913** [comment]
Thx @efstathiosntonas for a quick report after the v3.1.0, the fix is included in v3.1.1 🥳

**Comment by efstathiosntonas** [comment]
@mateusz1913 you're welcome 😄
